pipettes/firmware/CMakeLists.txt

```diff
@@ -29,7 +34,9 @@ add_executable(pipettes
 
 target_link_libraries(pipettes
         PUBLIC STM32G491RETx
-        STM32G4xx_Drivers_Pipettes STM32G4xx_FreeRTOS_Pipettes)
+        STM32G4xx_Drivers_Pipettes STM32G4xx_FreeRTOS_Pipettes
```

Comment:
I was kind of under the impression that the firmware part of CAN would also be a static library (like can-core). Is the reason we can't do this because of FreeRTOS and/or the HAL?

Reply:
I hoped to do that and had some success, but in the end decided to follow your pattern with the firmware files in `common`.